app/models/invenio_rdm_record_converter.rb

```diff
@@ -231,17 +231,19 @@ def build_creator_contributor_json(creator)
   # return array of invenio formatted subjects
   def subjects_for_scheme(terms, scheme)
     if scheme != :tag
-      terms.map do |term|
+      mapped_terms = terms.map do |term|
         pid = @@header_lookup.pid_lookup_by_scheme(term, scheme)
 
         if pid.present?
-          {subject: term, identifier: pid, scheme: scheme}
+          {id: pid}
         else
-          {subject: "#{term}: DigitalHub field #{scheme}"}
+          nil
         end
       end
+
+      mapped_terms.compact
     else
-      terms.map{ |term| {subject: term} }
```

Comment:
I can't find a note of it, but I thought we were adding tags? I will check with our team to confirm.